src/main/java/frc/robot/subsystems/Tabs/TabDriveBase.java

```diff
@@ -4,7 +4,9 @@
 import edu.wpi.first.networktables.*;
```

Comment:
More of a general complaint, but this is the only subsystem that uses TabContainers instead of SmartDashboard and/or Shuffleboard directly. The inconsistency makes it harder to maintain.

Reply:
We might look to disregard these tabs when competition comes around and standardized our output of data with everyone else. For now it's mainly for our testing purposes.